code/dqe/nullcount.q

```diff
@@ -0,0 +1,8 @@
+\d .dqe
+
+/- Given a table name as a symbol (tn), returns the number of nulls in tn.
+/- Works on partitioned tables in an hdb
+nullcount:{[tn]
```

Comment:
Do we need to also consider columns as possibility here?
Could add column and where null it does whole table?

Reply:
If we return a dictionary with multiple things to the DQE, will it add them as multiple entries in the resultstab table? If so, we could do this easily. Or would we have to call this function once for each column?